deny.toml

```diff
@@ -89,6 +89,12 @@ wildcards = "warn"
 # * all - Both lowest-version and simplest-path are used
 highlight = "all"
 
+# List of crates to deny
+deny = [
+  # scaffold used openssl for some reason
+  { name = "openssl-sys", wrappers = ["apollo-router-scaffold"] },
+]
+
```

Comment:
As an update, this **didn't work**, however other possibilities exist:

1. Add the `--exclude` flag to the actual invocation of `cargo deny` and specify `apollo-router-scaffold` there.  This should still have the effect of preventing `openssl` in the **rest** of this workspace tree — which is the most important thing, perhaps? 
2. Making this change to `wrappers`, which makes it only valid to have `openssl-sys` within the packages that immediately depend on it (because that's [how `wrappers` works](https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html?highlight=wrappers#the-wrappers-field-optional) (how this ever worked, I just don't understand).
 
	```suggestion
	# List of crates to deny
	deny = [
	# cargo-scaffold uses git2 which uses libssh2-sys and libgit2-sys, which both require
	# openssl-sys. HOWEVER, this rule alone isn't "safe" since this means that we could
	# introduce `git2` in `apollo-router` and it would just be allowed.  That's bad.
	# To prevent that, we add another rule just after which only allows `git2` within
	# `cargo-scaffold`.  This is a bit wonky, but does work.
        { name = "openssl-sys", wrappers = ["git2", "libssh2-sys", "libgit2-sys"] },
        # note this rule is required to support the above exception.
        { name = "git2", wrappers = ["cargo-scaffold"] },
	]
	
	```

Reply:
Applied this, with a bit more and force-pushed.